docs/lib/datastore/fragments/android/getting-started/70_querySnippet.md

```diff
@@ -0,0 +1,22 @@
+<amplify-block-switcher>
+<amplify-block name="Java">
+
+```java
+Amplify.DataStore.query(Post.class,
+    result -> Log.i("MyAmplifyApp", "Posts retrieved successfully"),
+    error -> Log.e("MyAmplifyApp",  "Error retrieving posts", error)
+);
+```
+
+</amplify-block>
+<amplify-block name="Kotlin">
+
+```kotlin
+Amplify.DataStore.query(Post::class.java,
+    { result -> Log.i("MyAmplifyApp", "Posts retrieved successfully") },
```

Comment:
Drop `result ->`

Reply:
^ I think we should probably do this as a preferred style in all of our Kotlin examples, for the purpose of saving columns.

Reply:
shouldn't we actually at least print the `result`? I expected the result to be always relevant in `query()`.

Reply:
Oh, sure, I agree with that. My drop comment was on a scan and noticing it wasn't being used.

Reply:
^ Same here